src/vma/dev/ring_tap.h

```diff
@@ -40,7 +40,7 @@
 class ring_tap : public ring_slave
 {
 public:
-	ring_tap(int if_index, ring* parent = NULL);
+	ring_tap(int if_index, ring* parent = NULL, int fd = -1);
```

Comment:
I dont see any reason to use default values here. we should also remove parent default value